jetbrains-core/src/software/aws/toolkits/jetbrains/services/schemas/search/SchemaSearchDialog.kt

```diff
@@ -55,32 +46,23 @@ class SchemaSearchSingleRegistryDialog(
 
 class SchemaSearchAllRegistriesDialog(
     project: Project,
-    onCancelCallback: (SchemaSearchAllRegistriesDialogState) -> Unit,
+    onCancelCallback: (SchemaSearchDialogState) -> Unit,
```

Comment:
nit: having the cancel callback in the middle is weird (even though it was already there), can we move it?